src/System.Composition.TypedParts/src/Configurations.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
Given the build error, it seems like we still need to support `netstandard`(2.0) in this project. We could harvest the netstandard2.0 assets into the package, or use ifdefs for this code and use `HashHelpers` whenever `TargetGroup` == `netstandard` -- @ericstj thoughts on harvesting? It seems like this package is compat package from core to desktop and that we don't plan to add new API to it and like we don't service it much.